.eslintrc.js

```diff
@@ -41,7 +41,6 @@ module.exports = {
     "import/prefer-default-export": OFF,
     "simple-import-sort/imports": ERROR,
     "simple-import-sort/exports": ERROR,
-    "no-console": ["error", { allow: ["error"] }],
```

Comment:
I would advice against removing these rules, if we need a console statement somewhere in the codebase, we can ignore the rule inline.